### PR TITLE
checks on full stops in about_py.rst

### PR DESCRIPTION
**Pull request by AnjuJoon**

### CODE REVIEW

File: source/rst/about_py.rst [hunk 1]
```diff
@@ -53,7 +53,7 @@ Common Uses
 
 * multimedia, data processing, security, etc., etc., etc.
 
-Used extensively by Internet service and high tech companies such as
+Used extensively by Internet service and high tech companies such as:
```
Comment (jstac) [inline source/rst/about_py.rst]: Hi @AnjuJoon , to tell the truth I prefer it without this colon.  I prefer that, if the sentence is basically OK, then we shouldn't change it.

File: source/rst/about_py.rst [hunk 2]
```diff
@@ -147,7 +147,7 @@ Scientific Programming
 
 Python has become one of the core languages of scientific computing.
 
-It's either the dominant player or a major player in
+It's either the dominant player or a major player in-
```
Comment (jstac) [inline source/rst/about_py.rst]: Again, I prefer that we don't make this change, since it's not really necessary.

File: source/rst/about_py.rst [hunk 3]
```diff
@@ -217,7 +217,7 @@ For example, let's calculate :math:`\int_{-2}^2 \phi(z) dz` where :math:`\phi` i
     value
 
 
-SciPy includes many of the standard routines used in
+SciPy includes many of the standard routines used in:
```
Comment (jstac) [inline source/rst/about_py.rst]: Same.  I prefer without.

**Comment by AnjuJoon** [comment]
@jstac thankyou so much for the feedback. Even I prefer it the way you said.

**Comment by AnjuJoon** [comment]
@jstac I was about to reverse some of the changes in this, Should I do that now? I am confused since you merged it.

**Comment by jstac** [comment]
Ahh!  Sorry `:-)`

I just merged it by mistake!  Let me try to undo it.

**Pull request merged**